### PR TITLE
App: experimental features in beta only

### PR DESCRIPTION
**Pull request by lavocatt**
When the unleash switch `image-builder.new-wizard.enabled` will get activated on stage the blueprint and new wizard will be visible on the platform.
To make them only visible in preview we need to change the condition to check if Beta is selected by the user.

### CODE REVIEW

File: src/Components/ImagesTable/ImagesTable.tsx [hunk 1]
```diff
@@ -468,8 +469,10 @@ const Row = ({
 }: RowPropTypes) => {
   const [isExpanded, setIsExpanded] = useState(false);
   const handleToggle = () => setIsExpanded(!isExpanded);
+  const { isBeta } = useGetEnvironment();
   const experimentalFlag =
-    useFlag('image-builder.new-wizard.enabled') || process.env.EXPERIMENTAL;
+    (useFlag('image-builder.new-wizard.enabled') || process.env.EXPERIMENTAL) &&
+    isBeta();
```
Comment (ezr-ondrej) [inline src/Components/ImagesTable/ImagesTable.tsx:475]: I'd prefer `isPreview()`, but I guess we don't have that in our wrappers, right?

**Review by ezr-ondrej — COMMENTED** [review]
We'll need to update our tests to say we are in preview, currently we are using just the flag, but apart of that an one nit, nice work! :orange_heart:

**Comment by lucasgarfield** [comment]
Tests should be passing now. :smile:

**Comment by lucasgarfield** [comment]
Just noticed we missed one of the `const experimentalFlag` variables and that somehow we've also introduced a Typing bug so the PR check is failing.

We really should probably consolidate all of these into a single variable.

**Comment by ezr-ondrej** [comment]
> We really should probably consolidate all of these into a single variable.

Depends where we are on the decision for the wizard, but might make sense to have that var renamed from experimental to blueprint (or similar) and store it in the [Blueprint slice](https://github.com/osbuild/image-builder-frontend/pull/1712) as `enabled`, WDYT?

**Comment by lucasgarfield** [comment]
Ugh tests still failing, looking into it...

**Review by lucasgarfield — APPROVED** [review]
Passing all tests, going to merge. We will still need to switch the flag value in Unleash to see the new features in stage-preview.

**Pull request merged**